docs/nidigital/class.rst

```diff
@@ -1475,41 +1475,12 @@ get_pattern_name
 
 
 
-get_pattern_pin_indexes
------------------------
-
-    .. py:currentmodule:: nidigital.Session
-
-    .. py:method:: get_pattern_pin_indexes(start_label)
-
-            TBD
-
-            
-
-
-
-            :param start_label:
-
-
-                
-
-
-            :type start_label: str
-
-            :rtype: list of int
-            :return:
-
-
-                    
-
-
-
-get_pattern_pin_list
---------------------
+get_pattern_pin_names
+---------------------
 
     .. py:currentmodule:: nidigital.Session
 
-    .. py:method:: get_pattern_pin_list(start_label)
+    .. py:method:: get_pattern_pin_names(start_label)
```

Comment:
I think we can keep the original name that maps to LabVIEW and C APIs.
`get_pattern_pin_list`
Why change it?

Reply:
`get_pattern_pin_names` *is* the name of the function that returns pins as a list of strings and C API does not have a function that does that.